rust/operator-binary/src/controller.rs

```diff
@@ -220,10 +224,22 @@ pub enum Error {
         source: crate::product_logging::Error,
         cm_name: String,
     },
+    #[snafu(display("failed to patch service account [{APP_NAME}-sa]"))]
+    ApplyServiceAccount {
+        source: stackable_operator::error::Error,
+    },
+    #[snafu(display("failed to patch role binding [{APP_NAME}-rolebinding]"))]
+    ApplyRoleBinding {
+        source: stackable_operator::error::Error,
+    },
```

Comment:
The name will show up in the propagated errors.
```suggestion
    #[snafu(display("failed to patch service account"))]
    ApplyServiceAccount {
        source: stackable_operator::error::Error,
    },
    #[snafu(display("failed to patch role binding"))]
    ApplyRoleBinding {
        source: stackable_operator::error::Error,
    },
```